### PR TITLE
bugfix/fwf-4035:Hid sidebar in preview page

### PR DESCRIPTION
**Pull request by fahad-aot**
# Issue Tracking

JIRA:  https://aottech.atlassian.net/browse/FWF-4035
Issue Type: BUG/ FEATURE

# Changes
1. Hid sidebar in preview page  

# screenshots 
image

### CODE REVIEW

**Review by Bonymol-aot — APPROVED** [review]

File: forms-flow-theme/scss/_variables.scss [hunk 1]
```diff
@@ -202,6 +202,9 @@ hr {
   margin-top: 0;
   height: 100vh !important;
 }
+.margin-0{
+  margin: 0 !important;
+}
```
Comment (arun-s-aot) [inline forms-flow-theme/scss/_variables.scss:207]: can we remove this and instead use 'm-0' bootstrap class to html where margin-0 is used @fahad-aot

Comment (fahad-aot) [reply forms-flow-theme/scss/_variables.scss]: Updated

File: forms-flow-nav/src/root.component.js [hunk 2]
```diff
@@ -13,12 +13,20 @@ export default function Root(props) {
...
   useEffect(() => {
     const headerHeight = headerRef.current?.offsetHeight || 0;
     const totalHeight = `calc(100% - ${headerHeight}px)`;
     setSidenavHeight(totalHeight);
   }, [ hasMultitenancyHeader ]); 
 
+  useEffect(()=>{
+    const location = window.location.pathname;
+    if(location.includes("view-edit")){
+      setIsPreviewRoute(true);
```
Comment (arun-s-aot) [inline forms-flow-nav/src/root.component.js:27]: pls change to set

Comment (fahad-aot) [reply forms-flow-nav/src/root.component.js]: updated.

**Review by arun-s-aot — APPROVED** [review]

**Pull request merged**